docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,150 @@
+=========================
+Playing with your command
+=========================
+
+Now that you have started :doc:`your first plugin <first-plugin>`, maybe you
+want to write a more interesting command than the basic ``.hello`` one. Not
+that there is anything wrong with that command! The emoticons plugin is
+composed of commands like this one::
+
+    @plugin.command('shrug')
+    @plugin.action_command('shrugs')
+    def shrug(bot, trigger):
+        bot.say('¯\\_(ツ)_/¯')
+
+Which is one of the maintainers' favorite commands to use. However, let's see
+if we can do something a bit more *complex* than that.
+
+
+Greeting a user by their name
+=============================
+
+Have you noticed that a :ref:`plugin callable <Plugin callables>` takes **two
+arguments?** The first one is the ``bot``, i.e. an instance of Sopel (wrapped
+for your convenience) that you can use to
```

Comment:
I don't think it's worth mentioning the wrapper implementation detail here, the newbie reader doesn't really need to know about it. I think the `i.e.` can be dropped as well, although I _am_ fond of literary abbreviations (cf. half of what I say on IRC)

```suggestion
arguments?** The first one is the ``bot``, an instance of Sopel that you can
use to
```

Reply:
Also we're on track to eliminate the wrapper soon enough, anyway (see #2460).

@SnoopJ's suggestion leaves room for the `:doc:` xref below to be moved onto this line, I'm 98% sure (on mobile rn).

Reply:
Hm... yeah, that makes sense to me.